internal/jstest/jstest.go

```diff
@@ -85,8 +85,14 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 		out  []byte
 	}
 
-	// tokens is a counting semaphore used to enforce a limit of 20 concurrent calls to runShellWithScript.
-	tokens := make(chan struct{}, 20)
+	tokens := 50
+
+	t := os.Getenv("NUM_TOKENS")
```

Comment:
It is an internal library package, it should not read environment variables. Instead, we should use a flag in the main.go and pass it there